framework/widgets/LinkPager.php

```diff
@@ -217,8 +224,8 @@ protected function renderPageButton($label, $page, $class, $disabled, $active)
         }
         if ($disabled) {
             Html::addCssClass($options, $this->disabledPageCssClass);
-
-            return Html::tag('li', Html::tag('span', $label), $options);
+            $tag = ArrayHelper::remove($this->disabledPageElementOptions, 'tag', 'span');
+            return Html::tag('li', Html::tag($tag, $label, $this->disabledPageElementOptions), $options);
```

Comment:
Empty line before the `return` statement is required according to code style agreement